tests/spec/filer.filesystem.spec.js

```diff
@@ -1,16 +1,17 @@
-var Filer = require('../../src');
-var util = require('../lib/test-utils.js');
-var expect = require('chai').expect;
+'use strict';
+const Filer = require('../../src');
+const util = require('../lib/test-utils.js');
+const expect = require('chai').expect;
 
 describe('Filer.FileSystem', function() {
   beforeEach(util.setup);
   afterEach(util.cleanup);
 
   it('should properly mount new or existing filesystem', function(done) {
-    var provider = util.provider().provider;
+    let provider = util.provider().provider;
```

Comment:
This can be `const` since `provider` will never change.

Reply:
changed